spec/support/govuk_component_matchers.rb

```diff
@@ -106,6 +106,15 @@ module GovukComponentMatchers
     end
   end
 
+  matcher :have_span_caption do |text|
```

Comment:
```suggestion
  matcher :have_caption do |text|
```

⭐ for adding this helper, could we remove the "span" from the matcher name to reduce cognitive load?